tests/ResourceSyncingTest.php

```diff
@@ -108,6 +111,28 @@
     UpdateOrCreateSyncedResource::$scopeGetModelQuery = null;
 });
 
+test('multiple tenants can have users synced to a central resource', function() {
```

Comment:
The name doesn't make it very clear what exactly this test asserts.
1. What's the reason for it being added here - what was not being tested before?
2. Can you make the name more descriptive

Is it just trying to assert that one user can "belong to multiple tenants"? Do we not test that elsewhere or is this just a basic test that establishes that basic thing, while other tests test this as part of larger assertions?